cmd/helm/push.go

```diff
@@ -41,6 +41,16 @@ func newPushCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 		Short: "push a chart to remote",
 		Long:  pushDesc,
 		Args:  require.MinimumNArgs(2),
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			if len(args) == 0 {
+				// Do file completion for the chart file to push
+				return nil, cobra.ShellCompDirectiveDefault
```

Comment:
The documentation here https://helm.sh/docs/topics/registries/ mentions that "The push subcommand can only be used against .tgz files".  But I when I renamed a package created with helm to any other name, I was still able to push it.

Therefore, I did not limit the file completion to files ending with `.tgz`.

Reply:
Oh interesting. Perhaps we should limit this in `push` as a follow-up. Makes sense for now 👌